<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Ax12View</class>
 <widget class="QWidget" name="Ax12View">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1238</width>
    <height>652</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout" rowstretch="0,10,0">
   <property name="margin">
    <number>0</number>
   </property>
   <item row="0" column="2">
    <spacer name="verticalSpacer_2">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="1" column="0">
    <spacer name="horizontalSpacer_6">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>40</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="1" column="1" colspan="3">
    <layout class="QGridLayout" name="gridLayout_4">
     <property name="verticalSpacing">
      <number>0</number>
     </property>
     <property name="bottomMargin">
      <number>0</number>
     </property>
     <item row="2" column="0">
      <layout class="QHBoxLayout" name="horizontalLayout_2">
       <item>
        <widget class="QPushButton" name="btnRefreshPositions">
         <property name="text">
          <string>Refresh positions</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="btnLockAll">
         <property name="text">
          <string>Lock All</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="btnReleaseAll">
         <property name="text">
          <string>Release All</string>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="horizontalSpacer_5">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </item>
     <item row="1" column="0">
      <widget class="QScrollArea" name="scrollArea">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
         <horstretch>0</horstretch>
         <verstretch>10</verstretch>
        </sizepolicy>
       </property>
       <property name="minimumSize">
        <size>
         <width>800</width>
         <height>0</height>
        </size>
       </property>
       <property name="styleSheet">
        <string notr="true">QScrollArea{border: 0px;}
.QScrollArea, .QWidget { background-color: rgba(0, 0, 0, 0) }
</string>
       </property>
       <property name="horizontalScrollBarPolicy">
        <enum>Qt::ScrollBarAlwaysOff</enum>
       </property>
       <property name="widgetResizable">
        <bool>true</bool>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
       <widget class="QWidget" name="scrollAreaWidgetContents">
        <property name="geometry">
         <rect>
          <x>0</x>
          <y>0</y>
          <width>800</width>
          <height>454</height>
         </rect>
        </property>
        <property name="sizePolicy">
         <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="minimumSize">
         <size>
          <width>600</width>
          <height>0</height>
         </size>
        </property>
        <layout class="QVBoxLayout" name="verticalLayout">
         <property name="spacing">
          <number>0</number>
         </property>
         <property name="margin">
          <number>0</number>
         </property>
         <item>
          <widget class="QGroupBox" name="gbAx12">
           <layout class="QVBoxLayout" name="Ax12BoxLayout">
            <property name="spacing">
             <number>0</number>
            </property>
            <item>
             <spacer name="verticalSpacer_4">
              <property name="orientation">
               <enum>Qt::Vertical</enum>
              </property>
              <property name="sizeHint" stdset="0">
               <size>
                <width>20</width>
                <height>40</height>
               </size>
              </property>
             </spacer>
            </item>
            <item>
             <spacer name="verticalSpacer_3">
              <property name="orientation">
               <enum>Qt::Vertical</enum>
              </property>
              <property name="sizeHint" stdset="0">
               <size>
                <width>20</width>
                <height>40</height>
               </size>
              </property>
             </spacer>
            </item>
           </layout>
          </widget>
         </item>
        </layout>
       </widget>
      </widget>
     </item>
     <item row="1" column="2">
      <widget class="QGroupBox" name="groupBox">
       <property name="title">
        <string/>
       </property>
       <layout class="QGridLayout" name="gridLayout_2">
        <item row="0" column="0" colspan="2">
         <widget class="Tools::Ax12MovementEditor" name="movementEditor" native="true">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Preferred" vsizetype="Expanding">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
         </widget>
        </item>
        <item row="1" column="0">
         <widget class="QWidget" name="widgetRunMovement" native="true">
          <layout class="QHBoxLayout" name="horizontalLayout">
           <property name="margin">
            <number>0</number>
           </property>
           <item>
            <spacer name="horizontalSpacer_9">
             <property name="orientation">
              <enum>Qt::Horizontal</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>5</width>
               <height>20</height>
              </size>
             </property>
            </spacer>
           </item>
           <item>
            <widget class="QToolButton" name="btnRunMovement">
             <property name="minimumSize">
              <size>
               <width>100</width>
               <height>0</height>
              </size>
             </property>
             <property name="text">
              <string>Run Movement</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLabel" name="label">
             <property name="text">
              <string>at</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QDoubleSpinBox" name="spSpeedLimit">
             <property name="suffix">
              <string> rpm</string>
             </property>
             <property name="maximum">
              <double>114.000000000000000</double>
             </property>
             <property name="value">
              <double>110.000000000000000</double>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
        </item>
       </layout>
      </widget>
     </item>
     <item row="2" column="2">
      <layout class="QHBoxLayout" name="horizontalLayout_5">
       <property name="spacing">
        <number>2</number>
       </property>
       <item>
        <spacer name="horizontalSpacer">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <widget class="QPushButton" name="btnImport">
         <property name="text">
          <string>Import</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="btnExport">
         <property name="text">
          <string>Export</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="btnLoadMovements">
         <property name="text">
          <string>Reload</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="btnSaveMovements">
         <property name="text">
          <string>Save</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item row="1" column="1">
      <widget class="QPushButton" name="btnRecord">
       <property name="text">
        <string>Record -&gt;</string>
       </property>
      </widget>
     </item>
     <item row="0" column="0">
      <layout class="QGridLayout" name="gridLayout_3">
       <item row="0" column="0">
        <spacer name="horizontalSpacer_10">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item row="0" column="3">
        <spacer name="horizontalSpacer_11">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item row="0" column="2">
        <widget class="QPushButton" name="btnAddAx12">
         <property name="minimumSize">
          <size>
           <width>180</width>
           <height>60</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>180</width>
           <height>16777215</height>
          </size>
         </property>
         <property name="text">
          <string>Add Ax12</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
   <item row="1" column="4">
    <spacer name="horizontalSpacer_2">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>40</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="2" column="2">
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>Tools::Ax12MovementEditor</class>
   <extends>QWidget</extends>
   <header>Ax12MovementEditor.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
